config.ts

```diff
@@ -41,31 +41,31 @@ export class Config implements IConfig {
     public readonly appInsightsInstrumentationKey = process.env.APPINSIGHTS_INSTRUMENTATIONKEY;
     public readonly port: number = process.env.PORT as unknown as number || 54321;
     public readonly bypassCache: boolean = process.env.AT11_NO_CACHE === "true";
-    public readonly cacheExpiration = 2 * 60 * 60 * 1000; // 2h
+    public readonly cacheExpiration = 2 * 60 * 60; // 2h
     public readonly requestTimeout = 15 * 1000; // 15s
     public readonly parserTimeout = 15 * 1000; // 15s
     public readonly restaurants = new Map<string, ReadonlyArray<{ id: number, name: string, urlFactory: (date: Date) => string, parser: IParser}>>([
         ["einpark", [
-            { id: 1, name: "Clock Block", urlFactory: _ => "https://restauracie.sme.sk/restauracia/clock-block_8537-petrzalka_664/denne-menu", parser: new ClockBlock() },
-            { id: 2, name: "Derby Pub", urlFactory: _ => "https://www.derbypub.sk/menu/obedove-menu", parser: new DerbyPub() },
-            { id: 3, name: "MKM Restaurant", urlFactory: _ => "https://restauracie.sme.sk/restauracia/mkm-pizzeria_1742-petrzalka_664/denne-menu", parser: new MKMRestaurant() },
-            { id: 4, name: "Classic restaurant & pub", urlFactory: _ => "https://restauracie.sme.sk/restauracia/classic-restaurant-pub_626-petrzalka_664/denne-menu", parser: new ClassicRestaurantPub() },
-            { id: 5, name: "Sídlisková pivárnička", urlFactory: _ => "https://menucka.sk/denne-menu/bratislava/sidliskova-pivarnicka", parser: new SidliskovaPivarnicka() }
+            { id: 1, name: "Clock Block", urlFactory: () => "https://restauracie.sme.sk/restauracia/clock-block_8537-petrzalka_664/denne-menu", parser: new ClockBlock() },

```

Comment:
the `()` indicates that  the lambda has no parameters, which is not true, it has one (date)
the  `_` is convention used to indicate that we dont care/use the parameter https://stackoverflow.com/questions/2778362/what-does-this-underscore-mean-in-lambda-expressions

Reply:
Yes you are right... but the eslint marked it as warning if I left there the "_ =>".
Also "() =>" is many times used as here when you don't want to use the parameters because it will then argue that you have unused args there.
In case of single parameter it is fine to use "_" but if you have arrow function with 2+ params then you cannot use "_" more than once, so you either define only those params that you will use or use "() =>" to ignore them.

But maybe the rules can be tweaked for this "_" so it will stop arguing, I will check that.

Also it might be beneficial even to ignore certain rules in this specific file, because if someone just want to add or update restaurant config then it will be easier to him to see that there is function with Date in it. So even not experienced dev will know about it, and also that's also one of my goals to have this simple and clear so anybody can start to contribute or fix the parsers in case they stop working.

Reply:
Good, it looks like there is ignore pattern:
`no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]`
I will apply it

Reply:
done